src/components/DappCard.js

```diff
@@ -48,7 +48,8 @@ export default function DappCard({ dapp, actions = true }) {
           {dapp.addedOn && (
```

Comment:
Not familiar with this codebase and haven't looked into the bug, but can you please explain why was this not working? Looks like `addedOn` is defined, based on line 48.

Reply:
AFAIK addedOn is set once when you first add a dapp to the list
So it should be the same as `dapp.skylinks[0]`
Getting the last item in that list should give you the date on which it was last updated

Reply:
Ah gotcha, that makes sense.